### PR TITLE
0.1.0

### PR DESCRIPTION
**Pull request by Arnatious**
Signed-off-by: Ted Kern <ted.kern@canonical.com>

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package ament_nodl
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+0.1.0 (2020-05-28)
+------------------
+* update doc to allow multiple nodl xml files
+* initial commit
```
Comment (kyrofa) [inline CHANGELOG.rst]: We can probably just start with "initial release" here, honestly.

Comment (Arnatious) [reply CHANGELOG.rst]: Fixed

File: README.md [hunk 2]
```diff
@@ -4,9 +4,10 @@ ament_nodl
...
 Usage
+
 ---
 With a file named `nodl.xml` in the same folder as your project's `CMakeLists.txt`, add the following line to your `CMakeLists.txt` before calling `ament_package()`
 
 ```CMake
 nodl_export_node_description_file(nodl.xml)
-```
\ No newline at end of file
+```
```
Comment (kyrofa) [inline README.md:13]: Does this file need to be part of this diff?

Comment (Arnatious) [reply README.md:13]: It'd be nice to have the markdowns cleaned up for release, it messies up this commit but its too small to warrant a PR of its own

Comment (kyrofa) [reply README.md:13]: Alright.

**Review by kyrofa — APPROVED** [review]

**Pull request merged**